Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -2120,14 +2113,8 @@ extension StatusTableViewController: CompletionDelegate {
 extension StatusTableViewController: PumpManagerStatusObserver {
     func pumpManager(_ pumpManager: PumpManager, didUpdate status: PumpManagerStatus, oldStatus: PumpManagerStatus) {
         log.default("PumpManager:%{public}@ did update status", String(describing: type(of: pumpManager)))
-        Task { @MainActor in
-
-            basalDeliveryState = status.basalDeliveryState
-            bolusState = status.bolusState
-
-            refreshContext.update(with: .status)
-            await self.reloadData(animated: true)
-        }
+        basalDeliveryState = status.basalDeliveryState
+        bolusState = status.bolusState
```

Comment:
This was calling `reloadData()` twice. First was during the update of the `bolusState` or `basalDeliveryState`.